test_install.sh

```diff
@@ -3,4 +3,8 @@
 timeout 20m pip install -vvvv chainer/dist/*.tar.gz --user
 
 # check if cupy is installed
-python -c 'import cupy'
+if [ $CUDNN = none ]; then
```

Comment:
How about making `$CUDNN` empty when you want to disable cuDNN and using following code?

```sh
if [ "$CUDNN" = none ]; then
```